CMakeLists.txt

```diff
@@ -190,49 +195,76 @@ ELSE()
 	MESSAGE(FATAL_ERROR "Please get them (www.python.org)")
 ENDIF()
 
-INCLUDE(FindSDL)
-IF(SDL_FOUND)
-	MESSAGE(STATUS "Looking for SDL1: found")
-ELSE()
-	MESSAGE(WARNING "Looking for SDL1: not found!")
+IF(SDL_BACKEND STREQUAL "SDL")
+	INCLUDE(FindSDL)
+	IF(USE_SDLMIXER)
+		FIND_PACKAGE(SDL_mixer MODULE)
+	ENDIF()
 ENDIF()
 
-# look for sdl2 if sdl1 was not found or a preference was stated
-IF(USE_SDL2 OR NOT SDL_FOUND)
+# FIXME: Compatibility - remove USE_SDL2 in the future
+IF((SDL_BACKEND STREQUAL "SDL2") OR USE_SDL2)
```

Comment:
Ok, but what if you have only sdl2 installed? That worked seamlessly before too.

And since you're not unsetting USE_SDL2, wouldn't switching back to sdl1 via the new option now not work properly (even before the latest commit)?